src/Application/src/CalculatorApplication.cpp

```diff
@@ -6,24 +6,28 @@
  */
 ////////////////////////////////////////////////////////////////////////////
 
+#include <chrono>
+#include <ctime>
 #include <future>
 #include <iostream>
 #include <limits>
 #include <sstream>
 #include <string>
 #include <thread>
 
+#include "AddCalculator.h"
```

Comment:
Do you need AddCalculator?

Try to be aware of your includes. You should keep them to the minimum. It can blow up file sizes and make it take longer to compile